src/main/java/org/cascadebot/cascadebot/commands/music/SeekCommand.java

```diff
@@ -0,0 +1,67 @@
+package org.cascadebot.cascadebot.commands.music;
+
+import com.ibm.icu.text.MeasureFormat;
+import com.ibm.icu.util.Measure;
+import com.ibm.icu.util.ULocale;
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.ParserUtils;
+import org.cascadebot.shared.Regex;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class SeekCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        String time = context.getMessage(0);
+
+        if (context.getArgs().length == 0) {
+            context.getUIMessaging().replyUsage();
+            return;
+        }
+
+        long millis;
+
+        if (context.getMusicPlayer().getPlayer().getPlayingTrack() == null) {
+            context.getTypedMessaging().replyDanger(context.i18n("commands.seek.no_music_playing"));
+            return;
+        }
+
+        if (context.isArgInteger(0)) {
+            millis = Long.parseLong(time);
```

Comment:
integer != long if you mush add a method in context for checking if an arg is a long

Reply:
This method should be renamed because it just used regex so it will be valid for long too

Reply:
We should probably have different methods and check the number in case someone does type a number out of the integer range.